src/utils/changes.ts

```diff
@@ -4,6 +4,9 @@ import { getVersion } from './version';
  * Path to the changelog file in the target repository
  */
 export const DEFAULT_CHANGELOG_PATH = 'CHANGELOG.md';
+export const DEFAULT_UNRELEASED_TITLE = 'Unreleased';
+export const DEFAULT_CHANGESET_BODY = '- No documented changes.';
+const HEADER_REGEX = /^(?:( *)## *([^\n]+?) *#*|^([^\n]+)\n *(?:-){2,}) *(?:\n+|$)/gm;
```

Comment:
Sharing regexes with `/g` can be dangerous since they remember state between executions. This is why it was previously instantiated in a function body rather than gobally. Are you sure that this does not misbehave?

Reply:
I did explicitly set its `lastIndex` property before each run that said there's no reason for this to be a module global. In an earlier iteration I was reusing this but then I abstracted that logic out into `locateChangeset`. Will move the regex back into the function.